organization/templates/organization/organization_confirm_delete.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout.html" %}
+
+{% load i18n %}
+{% load bootstrap3 %}
+
+{% block head_title %}{% trans "Delete Organization" %}: "{{ object }}"{% endblock %}
+
+{% block content %}
```

Comment:
Can you please indent this? If you see other pages, they are all indented.

Reply:
Do you talk about the content inside the `{% block content %}` or the `{% block head_title %}`:

```html
{% block head_title %}
    {% trans "Delete Organization" %}: "{{ object }}"
{% endblock %}
```

Or both :P?